css/intro.css

```diff
@@ -0,0 +1,82 @@
+.input-label-dark {
+  margin-left: 0.5rem;
+}
+
+.input-big {
+  height: 10.25rem;
+}
+
+#renderHeardAbout {
+  margin-bottom: 1rem;
+}
+
+#loading {
+  margin: 39vh 0;
+  text-align: center;
+}
+
+.not-found {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  min-height: 70vh;
+  margin: 3rem 0;
+}
+
+.not-found-text-h1 {
+  font-size: 1.75rem;
+  text-align: center;
+}
+
+.not-authorized {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  min-height: 70vh;
+  margin: 3rem 0;
+}
+
+.not-authorized-text-h1 {
+  font-size: 1.75rem;
+  text-align: center;
+}
+
+.hidden {
+  display: none;
+}
+
+@media screen and (max-width: 500px) {
+  .user-input {
+    font: 300 17px 'Roboto';
+    width: 22rem;
+    line-height: 32px;
+  }
+
+  .not-authorized-text-h1 {
```

Comment:
These styles can also be combined.

Reply:
Try not to use tag names in css classes names.
`.not-authorized-text` is enough.